templates/common/render/render.go

```diff
@@ -258,6 +266,7 @@ func RenderAlreadyDownloaded(ctx context.Context, dlMeta *templatesource.Downloa
 		rp:               p,
 		scope:            scope,
 		scratchDir:       scratchDir,
+		suppressPrint:    p.ManifestOnly, // if --manifest-only was given, then the user doesn't want printed output.
```

Comment:
Nit: it feels like "silent mode" is a feature that's valuable on it's own. Should we couple it to `manifest-only` like this?

Reply:
I could see that being valuable. But I think we can wait til somebody asks for it. I think it will be easy to uncouple in the future.